### PR TITLE
[plg_captcha_recaptcha_invisible] CS, missing filter

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

Just some CS and missing filter in manifest file.

### Testing Instructions

Code review.

### CODE REVIEW

File: plugins/captcha/recaptcha_invisible/recaptcha_invisible.php [hunk 1]
```diff
@@ -177,7 +178,7 @@ public function onSetupField(\Joomla\CMS\Form\Field\CaptchaField $field, \Simple
 	 * @param   string  $remoteip    The remote IP of the visitor.
 	 * @param   string  $response    The response received from Google.
 	 *
-	 * @return bool True if response is good | False if response is bad.
+	 * @return boolean True if response is good | False if response is bad.
```
Comment (Quy) [inline plugins/captcha/recaptcha_invisible/recaptcha_invisible.php]: Add a space before and after `boolean`. Add `boolean` to line 123.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ba3921f65723e4fcf873b798a7349db6dfabbc05<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22227">issues.joomla.org/tracker/joomla-cms/22227</a>.</sub>

**Review by mbabker — APPROVED** [review]

**Comment by twister65** [comment]
I have tested this item :white_check_mark: successfully on ba3921f65723e4fcf873b798a7349db6dfabbc05<br /><br />After applying this patch, reCaptcha invisible is displayed : 
				JPluginHelper::importPlugin('captcha', "recaptcha_invisible");
				$dispatcher = JEventDispatcher::getInstance();
				if ($dispatcher->trigger('onInit','dynamic_recaptcha_1') == true)
				{
					$output = $dispatcher->trigger('onDisplay', array("recaptcha", 'dynamic_recaptcha_1', 'class="g-recaptcha"'));
					echo $output[0];
				}<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22227">issues.joomla.org/tracker/joomla-cms/22227</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**